framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -394,6 +401,7 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
         this.visitorState = new VisitorState();
 
         this.supportedQuals = new HashSet<>();
+        this.supportedQualNames = new TreeSet<>();
```

Comment:
Is there a reason for using `TreeSet` here and a `HashSet` for `supportedQuals`? As this set will be queried a lot, `HashSet` should be better.